pkg/proc/proc_test.go

```diff
@@ -90,7 +90,15 @@ func withTestProcess(name string, t testing.TB, fn func(p *proc.Target, fixture
 	withTestProcessArgs(name, t, ".", []string{}, 0, fn)
 }
 
+func withTestProcessEnv(name string, t testing.TB, environ []string, fn func(p *proc.Target, fixture protest.Fixture)) {
```

Comment:
Just add an environment argument to `withTestProcessArgs` instead of adding a new function.